db/schema.rb

```diff
@@ -38,7 +38,7 @@
   add_index "carts", ["order_id"], name: "index_carts_on_order_id", using: :btree
   add_index "carts", ["user_id"], name: "index_carts_on_user_id", using: :btree
 
-  create_table "comments", force: true do |t|
+  create_table "comments", force: :cascade do |t|
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.